feature-libs/product-configurator/rulebased/styles/_configurator-product-title.scss

```diff
@@ -77,12 +88,26 @@
           // A larger value than the details need, this will be used to animate the closing/opening.
           // If the value is to large the closing will "delay" as the max-height will be reduced from the specified value down to 0.
           max-block-size: 400px;
+
+          @include forVersion(3.1) {
+            @include media-breakpoint-down(sm) {
+              padding-inline-start: 16px;
+            }
+
+            @include media-breakpoint-only(md) {
+              padding-inline-start: 32px;
+            }
+          }
         }
 
         .cx-details-image {
           margin-block-start: 15px;
           margin-block-end: 0px;
           max-inline-size: 100px;
+
+          @include forVersion(3.1) {
```

Comment:
change to 3.3 or add an todo

Reply:
The version will be updated accordingly, see #11238